tensorboard/uploader/exporter.py

```diff
@@ -52,12 +54,14 @@
 _FILENAME_METADATA = "metadata.json"
 # Output filename for scalar data within an experiment directory.
 _FILENAME_SCALARS = "scalars.json"
+_FILENAME_TENSORS = "tensors.json"
```

Comment:
Would be nice to have a comment about this file , similar to the one for scalars and tensors

Reply:
Done.